custom_components/plugwise-beta/binary_sensor.py

```diff
@@ -108,3 +130,64 @@ def _async_process_data(self):
                 self._icon = FLAME_ICON
 
         self.async_write_ha_state()
+
+
+class PwNotifySensor(SmileSensor, BinarySensorEntity):
```

Comment:
Is there a reason to fully redefine this instead of just overloading PwBinarySensor (`class PwNotifySensor(PwBinarySensor, BinarySensorEntity):`)? Then all PwNotifySensor needs is the `def device_state_attributes` that binary doesn't and the overloaded `def _async_process_data`.

Reply:
I was not aware that this was possible, now that I think of it...
I will give it a try...

Reply:
Working.